src/agent/tools/output_tools.py

```
@@ -21,22 +31,22 @@ def add_output_variable(
+                    "key_value": key_value,
+                    "variable_name": variable_name,
+                    "reporting_frequency": reporting_frequency,
                 }
             )
-        except Exception as exc:
-            return json.dumps({"success": False, "message": f"validation error: {exc}"})
-        config.output_variable.append(ov)
-        return json.dumps({"success": True, "message": f"added {variable_name}"})
+            idf.add(output_variable)
+            return _ok(f"Output:Variable '{variable_name}' added successfully.")
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)